### PR TITLE
[Fix] PHP warning message when using (AnsPress) Question Stats widget

### PR DESCRIPTION
**Pull request by mebishalnapit**
This PR includes:

* Fixes PHP Warning: Attempt to read property "answers" on bool when viewing non question page
* Fixes PHP Warning: Attempt to read property "post_modified_gmt" on bool when viewing non question page

### CODE REVIEW

File: includes/qaquery.php [hunk 1]
```diff
@@ -372,11 +372,14 @@ function ap_author_avatar( $size = 45, $_post = null ) {
  * Return total published answer count.
  *
  * @param  mixed $_post Post ID, Object or null.
- * @return integer
+ * @return integer|null
  */
 function ap_get_answers_count( $_post = null ) {
 	$_post = ap_get_post( $_post );
-	return (int) $_post->answers;
+	if ( $_post ) {
```
Comment (rahularyan) [inline includes/qaquery.php:379]: @mebishalnapit Also add check for `$_post->answers`.

**Review by rahularyan — CHANGES_REQUESTED** [review]

File: includes/qaquery.php [hunk 2]
```diff
@@ -501,13 +504,16 @@ function ap_recent_post_activity() {
  * Get last active time in human readable format.
  *
  * @param  mixed $post_id Post ID/Object.
- * @return string
+ * @return string|null
```
Comment (rahularyan) [inline includes/qaquery.php:507]: @mebishalnapit You cannot change the return type like this as this function may be used in other places or other peoples template customizations. This function should always return a string. When no post return a string `Invalid post`.

**Review by rahularyan — CHANGES_REQUESTED** [review]

File: includes/qaquery.php (hunk 1 above)
Comment (mebishalnapit) [reply includes/qaquery.php]: @rahularyan Additional check added.

File: includes/qaquery.php (hunk 2 above)
Comment (mebishalnapit) [reply includes/qaquery.php]: @rahularyan Modified to return `Invalid post` if no post is available.

**Review by rahularyan — APPROVED** [review]

**Review by rahularyan — APPROVED** [review]

**Pull request merged**